core/src/std_packages.rs

```diff
@@ -176,44 +289,172 @@ pub fn native_block_content<T, U>(
     Ok(Element::Compound(elements))
 }
 
-/// Example function for pushing a string to a list
-pub fn native_push_list<T, U>(
+/// Helper function to create text elements
+fn text_element(contents: String, id: GranularId) -> Element {
+    Element::Module {
+        name: "__text".to_string(),
+        args: Default::default(),
+        body: contents,
+        inline: true,
+        id: id,
```

Comment:
This can be just `id`:

```suggestion
        id,
```